Consul/Client.cs

```diff
@@ -148,6 +151,12 @@ public ConsulClientConfiguration()
             UriBuilder consulAddress = new UriBuilder("http://127.0.0.1:8500");
             ConfigureFromEnvironment(consulAddress);
             Address = consulAddress.Uri;
+
+            string ns = Environment.GetEnvironmentVariable("CONSUL_NAMESPACE");
```

Comment:
For the tests, should this be set like other env variables in https://github.com/G-Research/consuldotnet/blob/14beaea1098ec47300ce8e01dac8fb1e0fa6c379/Consul.Test/ClientTest.cs#L34?

Reply:
Guess that's not necessarily needed.

Reply:
> For the tests, should this be set like other env variables in
> 
> https://github.com/G-Research/consuldotnet/blob/14beaea1098ec47300ce8e01dac8fb1e0fa6c379/Consul.Test/ClientTest.cs#L34
> ?

I am not sure what you mean.

Reply:
nevermind 👍